packages/kit/src/background/services/ServiceHardware.ts

```diff
@@ -532,34 +532,23 @@ class ServiceHardware extends ServiceBase {
         break;
     }
 
+    // dev
+    const settings: { devMode: any } =
+      this.backgroundApi.appSelector((s) => s.settings) || {};
+    const { enable, updateDeviceSys } = settings.devMode || {};
+    const neverUpgrade = !!enable && !!updateDeviceSys;
```

Comment:
```suggestion
    const forceUpgrade = !!enable && !!updateDeviceSys;
```

在 dev 模式下开启固件更新开关会强制更新，从语义上来说感觉这里 `forceUpgrade` 更合适。

`neverUpgrade` 有点奇怪

Reply:
forceUpgrade 不太行。改成 alwaysUpgrade 吧。

Reply:
可以的